kyaml/fn/runtime/container/container.go

```diff
@@ -143,15 +144,21 @@ func (c *Filter) Filter(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
 	return c.Exec.Filter(nodes)
 }
 
-func (c *Filter) setupExec() {
+func (c *Filter) setupExec() error {
```

Comment:
The caller at L143 above isn't handling this new error

Reply:
thanks for the catch!